concepts/observability-strategy/strategy.md

```diff
@@ -103,17 +114,39 @@ All concepts follow general rules and will provide harmonized user APIs:
 - **Custom output protocols can be integrated**: Users can have custom transformation from the supported protocol, usually with additional operational effort (like running a dedicated Otel Collector sidecar or deployment in the users' Namespace). Furthermore, Kyma offers typical output plugins of the used collector technology as part of a pipeline definition; however, only with limited support.
 - **Blueprints are fully integrated**: Blueprints for local backend deployments are integrated with the used pipeline mechanism, so that they work instantly.
 
+## Integration Options
+
+With the new Observability architecture in place, the in-cluster backends won't be integrated by default out-of-the box. Instead, users will have the following three options to start their journey: Direct integration, indirect integration, and custom integration:
+
+### Direct Integration
+
+Kyma will provide a vendor-neutral integration point for traces, metrics, and logs, mainly using the OpenTelemetry Protocol (OTLP). To set up a new destination and select the data to be sent (for example, from specific Namespaces only), users can provide a custom configuration - a Kubernetes resource for a Kyma CRD. The system to integrate must support OTLP natively and can be either an external managed offering from a different vendor, or a custom system operated externally or even in-cluster. To get started quickly, Kyma will provide a blueprint to get the classic in-cluster setup deployed.
+
+![Direct Integration](./assets/integration-direct.drawio.svg)
+
+### Indirect Integration
+
+If the system to integrate with does not provide the vendor-neutral protocol natively, a converter is needed. Typically, this requires running a collector that supports OTLP and is configured as target for the direct integration. That collector then takes care of the conversion into the custom protocol and integration with the target system. Usually, that task will be realized by a custom Otel Collector instance.
+
+![Indirect Integration](./assets/integration-indirect.drawio.svg)
+
+### Custom Integration
+
+If users find the configuration options of the collector layer still too limiting, they can bring their own collector setup and disable the respective Kyma component for a signal type (logs, traces, metrics) so that it won't consume resources. With such custom setup, users get full  flexibility but also full responsibility. Users can also bring their own APM tooling at any time.
```

Comment:
```suggestion
If users find the configuration options of the collector layer still too limiting, they can bring their own collector setup and disable the respective Kyma component for a signal type (logs, traces, metrics) so that it won't consume resources. With such custom setup, users get full flexibility but also full responsibility. Users can also bring their own APM tooling at any time.
```